src/components/StringFormatter/StringFormatter.js

```diff
@@ -58,21 +58,21 @@ StringFormatter.propTypes = {
   /** @type {number} Number of lines to clamp value. */
   lines: number,
 
-  /** @type {string} The direction of the tooltip defintion. */
-  tooltipDirection: string,
+  /** Specify the direction of the tooltip. Can be either top or bottom. */
+  tooltipDirection: oneOf(['top', 'bottom']),
```

Comment:
There's a reason we replicate the `propType` instead of using `TooltipDefinition.propTypes.direction`, right? Is it because Storybook doesn't output the correct value otherwise?

Reply:
I'm not sure I follow but I can look into this 🤔 
I believe I was the one who originally added this prop, when I added tooltips to the `StringFormatter`, and so the `string` vs `oneOf` thing was a mistake, I think. But let me dig into this really quick.

Reply:
Oh yeah @SimonFinney just confirmed that when you use `TooltipDefinition.propTypes.direction`, storybook does not display correctly... (it shows the `propType` for the `direction` prop to be `custom` instead of `oneOf 'top' | 'bottom'` 👍

Reply:
Perfect - thanks for checking!